meetings/2022-01-13.md

```diff
@@ -0,0 +1,100 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2022-01-13
+
+## Links
+
+* **Recording**:  <https://youtu.be/qOzQysk2bCo>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1148>
+
+## Present
+
+* Antoine du Hamel @aduh95 (TSC)
+* Beth Griggs @BethGriggs (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* Gireesh Punathil @gireeshpunathil (TSC)
+* James Snell @jasnell (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Richard Lau @richardlau (TSC)
+* Michaël Zasso @targos (TSC)
+* Tobias Nießen @tniessen (TSC)
+* Rich Trott @Trott (TSC)
+
+## Agenda
+
+### Announcements
+
+* OpenJS World CFP open, hope to see you there
+* Security releases Jan 10, if you have not already done, check them out
+* Renewed SSL certificates
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* No CPC updates, no meeting this week. Some work going on license verification
+* Work going on related to documenting the programs under the community fund
+
+### nodejs/node
+
+* doc: make contributing info more discoverable [#41408](https://github.com/nodejs/node/pull/41408)
+  * No objections
+
+* Managing feature requests [#41113](https://github.com/nodejs/node/issues/41113)
+  * Gireesh, often user outlines, but is not technical enough, but there are no takers
+
+* docs: Clarification around real world risks and use cases of VM module [#40718](https://github.com/nodejs/node/issues/40718)
+  * Continue to get reports from people using the VM module to make a sandbox, but we are not
+    aware of how they get that to work.
+  * We already doc that it is not safe to do that - <https://nodejs.org/dist/latest-v17.x/docs/api/vm.html#vm-executing-javascript>
+  * The question to TSC is, is the current text strong enough? If not how do we want to make the
+    text stronger.
+  * Rich is should be stronger.
+  * James, people keep doing it because they have a need. Anna had some some work in the
+    past along the lines of what is needed in the past.
+    * Sketching out designs, hope to get to it later in the year.
+  * Chalker - Propose that we say any combination of some JavaScript code as isolation is
+    unsafe.
+  * Consensus that it should not be used, people are happy to make text stronger, just needs
+    somebody to come up with the stronger text.
+  * Rich moving text might help.
+  * Michael, making it bigger ?
```

Comment:
```suggestion
  * Rich: it should be stronger.
  * James: people keep doing it because they have a need. Anna had some work in the
    past along the lines of what is needed in the past.
    * Sketching out designs, hope to get to it later in the year.
  * Chalker: Propose that we say any combination of some JavaScript code as isolation is
    unsafe.
  * Consensus that it should not be used, people are happy to make text stronger, just needs
    somebody to come up with the stronger text.
  * Rich: moving text might help.
  * Michael: making it bigger ?
```